target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -355,6 +359,10 @@ ath79_setup_macs()
 	devolo,magic-2-wifi)
 		label_mac=$(macaddr_add "$(mtd_get_mac_binary art 0x1002)" 3)
 		;;
+	dlink,dch-g020-a1)
+		lan_mac=$(mtd_get_mac_text "mp" 0x1)
```

Comment:
Just curious: really 0x1, not 0x0?

Reply:
it really starts at 0x1, the first byte is 0x30 and the two macs are separated by 0x00 at 0x12, no idea what kind of structure / file format the manufacturer meant to apply here; it is followed by e.g. the label PIN Code, default ESSID and PSK, the rest of mp is blank.